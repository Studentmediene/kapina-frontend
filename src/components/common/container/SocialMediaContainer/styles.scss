@import 'variables.scss';

.socialMediaContainer {
  margin: $social-media-button-margin;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
}
