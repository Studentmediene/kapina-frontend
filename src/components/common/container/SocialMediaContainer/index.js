import React from 'react';

import styles from './styles.scss';
import SocialMediaButton from 'components/common/button/SocialMediaButton';

const SocialMediaContainer = () => (
  <div className={styles.socialMediaContainer}>
    <SocialMediaButton
      link="https://www.facebook.com/radiorevolt.no/"
      image="/assets/facebook_logo.svg"
      text="Facebook"
    />
    <SocialMediaButton
      link="https://www.instagram.com/radiorevolt/"
      image="/assets/instagram_logo.svg"
      text="Instagram"
    />
  </div>
);

export default SocialMediaContainer;
