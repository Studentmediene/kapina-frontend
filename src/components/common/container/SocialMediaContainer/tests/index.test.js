import React from 'react';
import { shallow } from 'enzyme';

import SocialMediaContainer from '../';

describe('<SocialMediaContainer />', () => {
  it('renders correctly', () => {
    const tree = shallow(<SocialMediaContainer />);
    expect(tree).toMatchSnapshot();
  });
});
