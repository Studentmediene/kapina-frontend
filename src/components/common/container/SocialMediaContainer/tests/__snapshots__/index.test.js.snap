// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocialMediaContainer /> renders correctly 1`] = `
<div
  className="socialMediaContainer"
>
  <SocialMediaButton
    image="facebook.svg"
    link="https://www.facebook.com/radiorevolt.no/"
    text="Facebook"
  />
  <SocialMediaButton
    image="instagram.svg"
    link="https://www.instagram.com/radiorevolt/"
    text="Instagram"
  />
</div>
`;
